ludwig/schema/config/model_config.py

```diff
@@ -60,56 +64,6 @@
 DEFAULTS_MODULES = {NAME, COLUMN, PROC_COLUMN, TYPE, TIED, DEFAULT_VALIDATION_METRIC}
 
 
```

Comment:
Moved the below functions into `ludwig.schema.config.utils` since it was used by both the `ModelMetadata` object and the `ModelConfig` object.